test/parallel_api/ranges/transform2_ranges_factory_sycl.pass.cpp

```diff
@@ -13,10 +13,11 @@
 //
 //===----------------------------------------------------------------------===//
 
-#include <oneapi/dpl/execution>
-
 #include "support/test_config.h"
 
+#include _PSTL_TEST_HEADER(execution)
+#include _PSTL_TEST_HEADER(algorithm)
+
 #if _ENABLE_RANGES_TESTING
 #include <oneapi/dpl/ranges>
```

Comment:
If we are adjusting to use `_PSTL_TEST_HEADER` above, should we do the same for `ranges`?

Reply:
yea.. is not consistently.